_cloudstor/02-generate-cloudstor-password.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Generate App Password
+order: 2
+duration: 5
+---
+
+1. Go to [Cloudstor Dashboard](https://cloudstor.aarnet.edu.au/) and select the university or institution are you based. The login process is via AAF and is exactly the same as login to Nectar Cloud Dashboard.
+![Cloudstor Login Page]({{site.baseurl}}/assets/images/cloudstor/cloudstor-login-page.png)
+2. Click the `Settings` icon on the top right hand side to get into the `Settings` page. Then click `Security` menu item on the left hand side. 
+![Cloudstor Setting Page]({{site.baseurl}}/assets/images/cloudstor/cloudstor-app-password.png)
+3. On the App passwords page, type in an app name ((a good choice is the name of your Nectar Virtual Machine)) and click `Create new app password` button. Then save the generated username and password in your computer. You need to use this username and password later in the setup of CloudStor.
```

Comment:
`((a good choice is the name of your Nectar Virtual Machine))` <-- does this need double brackets?